src/toolbar/styles/toolbar.m.css

```diff
@@ -0,0 +1,59 @@
+@import '../../common/styles/variables.css';
+
+.actionsFixed {
+	display: flex;
+	flex: 1;
+	justify-content: flex-end;
+}
+
+.menuButtonFixed {
+	position: absolute;
+	right: 0;
+	top: 50%;
+	transform: translateY(-50%);
+}
+
+.rootFixed {
+	height: 100%;
+	left: 0;
+	overflow: hidden;
+	position: fixed;
+	top: 0;
+	width: 100%;
+	z-index: var(--zindex-fixed);
+}
+
+.toolbarFixed {
+	display: flex;
+	position: relative;
+}
+
+.stickyFixed .toolbarFixed {
+	flex-grow: 0;
+	flex-shrink: 0;
+}
+
+.stickyFixed {
+	display: flex;
+	flex-direction: column;
+}
+
+.titleFixed {
+	flex-grow: 0;
+	flex-shrink: 0;
+}
+
+.stickyFixed .contentFixed {
+	flex: 1;
+	overflow-y: auto;
+}
+
+.action { }
```

Comment:
We may not need all these classes to be themeable, but it felt right to expose as many of them as possible.